backend/apps/beaver_etl/beaver_etl/settings.py

```diff
@@ -17,6 +17,8 @@ class Settings(BaseSettings):
     service_url: str = "http://localhost:8000"
     chunk_size: int = 10
 
+    api_secret_token: str = "73092d292062a7cf2476d56770064c5d88b063d8"
```

Comment:
We store this token in the code for easier development, and it's a temporary solution, right?

Reply:
yep that is just for the local development it will be overwritten by env var

Reply:
I think you are right. It makes sense to remove it

Reply:
done